app/models/concerns/effective_devise_user.rb

```diff
@@ -162,6 +166,32 @@ def send_reset_password_instructions(attributes = {})
       recoverable
     end
 
+    # https://github.com/heartcombo/devise/blob/f6e73e5b5c8f519f4be29ac9069c6ed8a2343ce4/lib/devise/models/authenticatable.rb#L276
```

Comment:
Could we add fallback if the alternate_email field doesn't exist?

Just do the alternate email stuff on `respond_to?(:alternate_email)`

We can't assume every User will have one.